db/seeds/pages.rb

```diff
@@ -0,0 +1,166 @@
+Page.delete_all
+
+include PreBuiltPagesHelper
+
+pages = {}
+
+pages['TopPage'] = <<'EOS'
+徳田・村井・楠本・中村・高汐・バンミーター・植原・三次・中澤・武田合同研究プロジェクト(RG)のポータルサイトです。
+
+毎週木曜日の4限(2:45PM~4:15PM)と5限(4:30PM~6:00PM)にι23教室でRG全体授業を行っています。
+EOS
+
+pages['WipTerm'] = <<'EOS'
+# WIP/TERM
+## WIP (Work In Progress)
+RGに所属し始めてから2学期目以降の学生は、学期中に取り組んだ内容とそれを踏まえた今後の活動・研究の展望について発表するWIP発表が原則として必須になります。
+新規履修者の方は自由参加でWIPの発表ができます。
+
+なお、WIP発表は中間と最終の2回に分けて行われ、そのどちらで発表を行うかを選択することができます。
+
+## TERM
+RGで卒業プロジェクトを履修するためには、遅くとも卒業プロジェクト履修前の学期中にTERM発表を行い、教員からの合格をもらう必要があります。
+
+TERM発表ではWIP発表と異なり、中間発表で研究テーマ・評価方法・スケジュールについて話した上で、最終発表では実際に研究として取り組んだ結果の発表を求められます。
+EOS
+
+pages['Thesis'] = <<'EOS'
+# 卒論 / Graduate thesis
+RGの教員の卒業プロジェクトを履修する場合は前学期までにTERMプロジェクトで合格をもらう必要があります。
+
+## 卒業プロジェクト1 / Graduation Project 1
+
+学期中に論文テーマとスケジュールについて話す所信表明を行い、学期末に英語でポスター発表を行う必要があります。
+
+## 卒業プロジェクト2 / Graduation Project 2
+
+学事の規定する期限までに論文を提出して受理され、学期末に最終成果の発表を行って合格をもらう必要があります。
+EOS
+
+pages["#{term_name}"] = <<"EOS"
+# #{term_name}授業情報
+
+| 回 | 日付 | 授業内容 |
+|:---:|:---:|:---:|
+EOS
+
+Meeting.order(:start_at).each.with_index(1) do |meeting, i|
+  main = meeting.presentations.first
+  content = main ? main.title : "未定"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.